### PR TITLE
BEMHTML: should trim and escape cls

### PR DESCRIPTION
**Pull request by miripiruni**
Fix for #190

### CODE REVIEW

File: lib/bemhtml/index.js [hunk 1]
```diff
@@ -130,10 +130,10 @@ BEMHTML.prototype.render = function render(context,
     }
 
     if (cls)
-      out += ' ' + cls;
+      out += ' ' + (cls.trim ? utils.attrEscape(cls.trim()) : cls);
   } else {
     if (cls)
-      out += cls;
+      out += cls.trim ? utils.attrEscape(cls.trim()) : cls;
```
Comment (zxqfox) [inline lib/bemhtml/index.js]: Ээээээ? What the heck is that condition?

Comment (miripiruni) [inline lib/bemhtml/index.js]: @zxqfox I mean `trim()` exist only in Strings instances.

Comment (zxqfox) [inline lib/bemhtml/index.js]: Em. What else you can pass here?

Why not just to do `cls = utils.attrEscape(cls).trim();` above?

Comment (miripiruni) [inline lib/bemhtml/index.js]: @zxqfox why it’s matter?

Comment (zxqfox) [inline lib/bemhtml/index.js]: It's javascript. `{toString:()=>'">'}` will fall

**Comment by miripiruni** [comment]
@zxqfox updated.

**Comment by miripiruni** [comment]
Review needed.

 cc @zxqfox @tadatuta @veged

File: lib/bemhtml/index.js [hunk 2]
```diff
@@ -130,10 +130,10 @@ BEMHTML.prototype.render = function render(context,
     }
 
     if (cls)
-      out += ' ' + cls;
+      out += ' ' + (cls.trim ? utils.attrEscape(cls).trim() : cls);
```
Comment (tadatuta) [inline lib/bemhtml/index.js]: if you use assert for `cls.trim` just to check if it's an instance of String, why not to do it in more intuitive way: `typeof cls === 'string'`?

**Comment by tadatuta** [comment]
lgtm

**Pull request merged**